src/ring_theory/derivation.lean

```diff
@@ -440,3 +447,97 @@ begin
 end
 
 end to_square_zero
+
+section derivation_module
+
+open_locale tensor_product
+
+variables (R S : Type*) [comm_ring R] [comm_ring S] [algebra R S]
+
+/-- The kernel of the map `S ⊗[R] S →ₐ[R] S`. -/
```

Comment:
Is there exactly one map `S ⊗[R] S →ₐ[R] S`? If so, do we have a proof of that statement? If not, can you be explicit about which map?